apps/api/src/app/notification-groups/usecases/get-notification-group/get-notification-group.usecase.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { NotificationGroupRepository, NotificationGroupEntity } from '@novu/dal';
+import { GetNotificationGroupCommand } from './get-notification-group.command';
+
+@Injectable()
+export class GetNotificationGroup {
+  constructor(private notificationGroupRepository: NotificationGroupRepository) {}
+
+  async execute(command: GetNotificationGroupCommand): Promise<NotificationGroupEntity> {
+    const result = await this.notificationGroupRepository.findById(command.id);
+    if (result === null) throw new NotFoundException();
```

Comment:
Novu is a multi-tenant system hence we are required to use the `command.organizationId` or `command.environmentId` in the dal queries. Could you please update the query here.